public/css/custom/main.css

```diff
@@ -1,55 +1,122 @@
+/* Variables */
+
+:root {
+    --smoky_black: #090c08;
+    --eerie_black: #191919;
+    --rich_black: #080705;
+    --green_black: #07393c;
+    --cadet: #5c6b73;
+    --gunmetal: #253237;
+    --light_cyan: #e0fbfc;
+    --flame: #f15025;
+    --english_violet: #474056;
+    --red_violet: #912f40;
+    --platinum: #e6e8e6;
+    --charcoal: #40434e;
+    --charcoal_blue: #364958;
+    --baby_powder: #fffffa;
+    --gold: #ddbe46;
+    --middle_blue: #90ddf0;
+    --light_gray: #ced0ce;
+    --steel_teal: #55828b;
+    --outer_space: #343f3e;
+    --davys_grey: #505a5b;
+    --manatee: #8f91a2;
+}
+
 /* General styling */
 
 * {
     box-sizing: border-box;
 }
 
 body {
-    background-color: rgb(41, 41, 41);
+    /* background-color: rgb(41, 41, 41); */
+    background-color: var(--charcoal_blue);
+    display: grid;
+    /* grid-template-rows: 48px auto 48px; */
+    height: 100vh;
+    width: 100vw;
+    font-family: "Ubuntu";
 }
 
-p {
-    color: white;
+.body--nav {
+    /* background-color: rgb(41, 41, 41); */
+    display: grid;
+    grid-template-rows: 48px auto 48px;
+    height: 100vh;
+    width: 100vw;
 }
 
-.label--form {
-    color: white;
-    font-style: italic;
+p {
+    color: var(--baby_powder);
 }
 
 .content {
-    height: auto;
+    overflow: scroll;
+}
+
+footer {
+    height: 48px;
+}
+
+.logo--auth {
+    font-family: "Aclonica";
+    text-align: center;
+    font-size: 3em;
+    color: var(--platinum);
+}
+
+hr {
+    border-top: 1.5px solid var(--light_cyan);
+    margin-bottom: 30px;
 }
 
 /* Header styling */
 
 header {
+    font-family: "Aclonica";
     height: 48px;
-    background-color: black;
+    background-color: var(--gunmetal);
     margin-bottom: 12px;
 }
 
 .header--text {
     text-align: center;
     font-size: 2em;
-    color: white;
-    font: bold;
+    color: var(--platinum);
+    /* font: bold; */
```

Comment:
It's better to completely remove a line instead of commenting it out. If you need it later on you can still find it in your previous commits.